src/Shaders/ShadersInclude/helperFunctions.fx

```diff
@@ -73,6 +73,27 @@ float square(float value)
     return value * value;
 }
 
+// https://learnopengl.com/PBR/IBL/Specular-IBL
+// Hammersley
+float radicalInverse_VdC(uint bits) 

```

Comment:
This should be enclosed in #ifdef WEBGL2 and else use the fallback as defined here :

https://learnopengl.com/PBR/IBL/Specular-IBL

in the section Hammersley sequence without bit operator support

```

float VanDerCorpus(uint n, uint base)
{
    float invBase = 1.0 / float(base);
    float denom   = 1.0;
    float result  = 0.0;

    for(uint i = 0u; i < 32u; ++i)
    {
        if(n > 0u)
        {
            denom   = mod(float(n), 2.0);
            result += denom * invBase;
            invBase = invBase / 2.0;
            n       = uint(float(n) / 2.0);
        }
    }

    return result;
}
// ----------------------------------------------------------------------------
vec2 HammersleyNoBitOps(uint i, uint N)
{
    return vec2(float(i)/float(N), VanDerCorpus(i, 2u));
}
```